### PR TITLE
fix: add container selectors on flow-container page

### PR DESCRIPTION
**Pull request by konst8**
## Changes
Add container selectors on the flow-container page.

## Related Issues
https://github.com/johnsonandjohnson/Bodiless-JS/issues/1510

### CODE REVIEW

**Review by hvanyo — COMMENTED** [review]
@konst8  Can we try put the identifiers into flowcontainer component if possible?

**Review by dewen — APPROVED** [review]

**Comment by konst8** [comment]
> @konst8 Can we try put the identifiers into flowcontainer component if possible?

@hvanyo in my understanding, in order to do that we will have to add the "...rest" of props here:
https://github.com/johnsonandjohnson/Bodiless-JS/blob/main/packages/bodiless-layouts/src/FlowContainer/EditFlowContainer.tsx#L95
which may possibly cause some regression

File: sites/test-site/src/data/pages/flow-container/index.tsx [hunk 1]
```diff
@@ -112,84 +112,115 @@ const FlowContainerPage = (props: any) => (
...
-        getDefaultWidth={() => 'w-full lg:w-3/4'}
-      />
-      <H2>Contentful Flow Container</H2>
-      <ContentfulFlowContainer />
-      <H2>Limited Flow Container</H2>
-      <FlowContainerLimited nodeKey="limited" />
-      <H2>Nested Flow Container with Default Items</H2>
-      <RegionContainer nodeKey="region" />
+
+      <div aria-label="default-fc-wrapper">
```
Comment (hvanyo) [inline sites/test-site/src/data/pages/flow-container/index.tsx:116]: Let's just switch over id's for now.   This isn't a good case for using accessilbiity to identifier containers.

Comment (konst8) [reply sites/test-site/src/data/pages/flow-container/index.tsx]: Updated. Made the flowContainer allowing id prop, removed the wrapper with aria-label. 
Regarding the styleguide/flowcontainer page of __cxstarter__ site - it uses generic template and flow containers are added through the editor interface there, so it won't be easy to add the same ids there. As far as I can see.

**Review by hvanyo — APPROVED** [review]

File: packages/bodiless-layouts/src/FlowContainer/StaticFlowContainer.tsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ const flowContainerComponentStart: FlowContainerComponents = {
 
 const NodeProvider = withNode(React.Fragment);
 
-const StaticFlowContainer: FC<DesignableProps> = ({ design }) => {
+const StaticFlowContainer: FC<DesignableProps & { id?: string }> = ({ design, id }) => {
```
Comment (dewen) [inline packages/bodiless-layouts/src/FlowContainer/StaticFlowContainer.tsx:34]: shall we have a default value for `id`? since it's optional by type

Comment (konst8) [reply packages/bodiless-layouts/src/FlowContainer/StaticFlowContainer.tsx:34]: @dewen i think undefined is okay as default - the id will not be added to the element in this case, which is desirable behavior for us.

**Pull request closed without merging**